docs/_docs/reference/experimental/modularity.md

```diff
@@ -134,6 +126,38 @@ only if the class refers to a type member of `x`. But it turns out that this
 scheme is unimplementable since it would quickly lead to cyclic references
 when typechecking recursive class graphs. So an explicit `tracked` looks like the best available option.
 
+## Tracked members
+
+The `tracked` modifier can also be used for `val` members of classes and traits
+to force the type of the member (or it's overriding member) to be as exact as
+possible. More precisely, it will will assign the `tracked` member the infered
```

Comment:
@KacperFKorban 

Though it is merged, "it will will" should be "it will", right?